### ISSUE TITLE
Any news for an update?

### ISSUE THREAD

**Issue by MaartenSomay** [TherionRO/YouTubeiVanced#15]
Hey Orion, im wondering if you will still update the app to the latest uyou+. Since uyou+ has issues with altstore/sidestore, and yours not. I love using yours!

**Comment by FlayingPaw87** [comment]
Yes, I also love the project. Please don't throw in the towel 🙏.

**Comment by TherionRO** [comment]
Hello,im sorry for not making any updates. I had some health issues in real life,if you will check the activity of my github account you will see that I was mostly inactive for the past few months. A week ago I switched to Android again because I just hated everything about iOS and how programming and everything was. I will update iVanced to the latest youtube version but most likely the new features I was talking about won't come anymore,just the Google's new patches and the new plugins from the uYou+ developers,im sorry for any inconvenience made,iVanced will be updated very soon,maybe today.

**Comment by TherionRO** [comment]
iVanced was updated yesterday to the latest YouTube version and uYou!
